### DISCUSSION TITLE
Get list of undocumented statements

### DISCUSSION THREAD

**Discussion by EdwardMoyse** [compodoc/compodoc#1378] [General]
In our project we sometimes get PRs failing CI because the coverage is less than 100%. Is there any way to get a report on the undocumented "statements"?

As a concrete example, for the case given in the [coverage documentation](https://compodoc.app/guides/documentation-coverage.html), `MediaQueryService` is 60% documented. Is there a way to get a list of what is missing? Or easily show it in the html (scrolling through big classes can be pretty painful).

**Comment by vogloblinsky** [comment]
Yes, it could be an enhancement of this command : in the last line of the terminal output, get a list of failing files
